external-crates/move/crates/move-ir-to-bytecode-syntax/src/lexer.rs

```diff
@@ -229,14 +229,9 @@ impl<'input> Lexer<'input> {
                     match &text[len..].chars().next() {
                         Some('"') => {
                             // Special case for ByteArrayValue: h\"[0-9A-Fa-f]*\"
-                            let mut bvlen = 0;
-                            if name == "h" && {
-                                bvlen = get_byte_array_value_len(&text[(len + 1)..]);
```

Comment:
This version was not calling `get_byte_array_value_len` if `name` wasn't `h`, let me know if you preferred that